_posts/2022-05-16-opentelemetry-metrics-for-data-prepper.md

```diff
@@ -0,0 +1,114 @@
+---
+layout: post
+title:  "Metrics Ingestion with Data Prepper using OpenTelemetry"
+authors:
+  - kschnitter
+  - ksternad
+date:   2022-05-06 12:00:00 -0200
+categories:
+  - technical-post
+---
+Data Prepper offers great capabilities for ingesting traces and logs into OpenSearch.
+Until now, that leaves out the third pillar of observability: Metrics.
+Data Prepper 1.4.0 introduces support for metrics received via OpenTelemetry.
+This feature aligns nicely with the OpenTelemetry traces support in Data Prepper.
+
+## Why Ingest Metrics in OpenSearch?
+
+At a first glance, OpenSearch offers great support for storing and analyzing any structured observability data.
+It provides powerful search and aggregation capabilities.
+Excellent visualizations support users in generating insights.
+The Vega visualizations lift the metrics use-case to a whole new level.
```

Comment:
```suggestion
Custom-made, self-programmed Vega visualizations even may lift the metrics use-case to a whole new level.
```

Reply:
I also was going to comment on clarifying this section. It may also be to:
* Clarify how Vega sits in the architecture
* Link to the Vega project when you first mention it.

Reply:
The Vega feature is inherited from Kibana. I could not find OpenSearch documentation for Vega. But I can confirm it's working ;-)